README.md

```diff
@@ -232,6 +241,29 @@ for example with `fdisk` command, before writing **"ext4"** image.
 The USB-flash image content is the RPI 5 Linux rootfs based on RPI5 bsp yocto build
 [meta-raspberrypi](https://git.yoctoproject.org/meta-raspberrypi) with Xen tools enabled.
 
+### Flash rootfs image to the nvme storage
+
+* Create Sd-card with official Ubuntu from Raspberry foundation.
+* Copy file yocto/build-domd/tmp/deploy/images/raspberrypi5/rpi5-image-xt-domd-raspberrypi5.rootfs.ext to the USB-Flash dongle.
+* Boot from Sd card with Ubuntu. Plug USB-Flash dongle to the Raspberry Pi.
+* Create partition on nvme storage with command:
+```
+(
```

Comment:
this should be running from root. please mention this. sudo will not work.
Also please set Raspberry PI version explicitly because it's not clear if it's gonna work on the previous versions.

Reply:
`sudo` works. `sudo` should be before `fdisk` not before `echo`
Actually we are talking about rpi5 only. But it should work also on rpi4